### ISSUE TITLE
Bug - Duplicate song title overlay

### ISSUE THREAD

**Issue by laynetrain** [HermesApp/Hermes#315]
![image](https://user-images.githubusercontent.com/16389820/29579740-b6414016-8741-11e7-8547-7a1e95445770.png)

**Comment by laynetrain** [comment]
Version 1.3.1 (2058)

**Comment by nriley** [comment]
Yeah this is a fun one. This is a result of the arrows appearing on hover yet allowing the text to be selected - you're seeing both copies which unfortunately have different font metrics. This usually lines up a little better. What OS version and Mac, what display and have you changed your antialiasing settings in any way?

--Nicholas

**Comment by laynetrain** [comment]
It personally doesn't bother me, but I love the app so much I thought I'd
let someone know!

MacBook Pro (Retina, 15-inch, Mid 2015)
macOS Sierra 10.12.6 (16G29)

Settings > Display > [checked] Default resolution
Settings > General > [checked] Use LCD font smoothing when available




*Layne Sadler*layne.sadler@gmail.com
linkedin.com/in/laynesadler
978.833.4190

**Comment by nriley** [comment]
OK unfortunately I have not been able to reproduce this one. What's bizarre is how the word "Joe" seems to line up but nothing else.  If you can grab any more screen shots of songs that might do it, let me know.
